python3/ltk/actions/status_action.py

```diff
@@ -84,24 +84,42 @@ def _get_status_of_doc(self, doc_id, detailed):
             if check_response(response):
                 raise_error(response.json(), error_message, True, doc_id)
             else:
-                #if document has recently been added, determined by uploadWaitTime, let user know that document is still being processed
-                #otherwise, suggest that user check TMS for deletion or potential upload problems
-                entry = self.doc_manager.get_doc_by_prop('id', doc_id)
-                diff = time.time() - entry['added']
-                if diff < self.uploadWaitTime:
-                    error_message = "\'" +entry['file_name']+ "\' is still being processed"
-                    raise_error("", str(response.status_code)+" Not Found: "+error_message, True, doc_id)
-                else:
-                    error_message = "Check Lingotek TMS to see if \'" +entry['file_name']+ "\' has been deleted or was not properly imported"
-                    raise_error("", str(response.status_code)+" Not Found: "+error_message, True, doc_id)
+                self._get_process(entry)
         else:
             title = response.json()['properties']['title']
             progress = response.json()['properties']['progress']
             statustext = response.json()['properties']['status'].upper()
             self._print_status(title, doc_id, progress, statustext)
             if detailed:
                 self._print_detailed_status(doc_id, title)
-                
+
+    def _get_process(self, entry):
+        process_id = entry['process_id']
```

Comment:
Account for the entry not having the process id key

Reply:
Check for process id key has been added